test/interfaces/flags.test.ts

```diff
@@ -126,6 +126,8 @@ class MyCommand extends BaseCommand {
     defaultArrayFlag: arrayFlag({default: ['foo', 'bar']}),
   }
 
+  public static '--' = true
```

Comment:
What's that doing?

Reply:
It is enabling passthrough. I can add a comment

Reply:
If it weren't there, what would be different and why?

Reply:
asking because removing it didn't make any noticeable different, and no tests were referring to it.